uswds/widget-uswds-styles/_prefixed-usa-modal.scss

```diff
@@ -0,0 +1,181 @@
+@use "uswds-core" as * with (
+  $theme-modal-default-max-width: "tablet"
+);
```

Comment:
This changes the max width of the modal from 30rem to 40rem. Our previous setting was 35rem. The max width of the form element is 30rem and the modal looks weird if its max width is not bigger than the form's max width. I also tried the large version of the modal but it has too much padding, too much white space, IMO.